numpyro/distributions/util.py

```diff
@@ -174,14 +174,24 @@ def _standard_gamma_grad(sample, alpha):
     return grads.reshape(alpha.shape)
 
 
+def _standard_gamma_batching_rule(batched_args, batch_dims):
+    x, y = batched_args
+    bx, by = batch_dims
+    size = next(t.shape[i] for t, i in zip(batched_args, batch_dims)
```

Comment:
This seems to only use information from `x` to decide the size. Do we not need to broadcast with `y[i]`?

Reply:
It is required by vmap that batching sizes must be equal for vmap args, so we dont need to check the batch size of y if bx is not None.